ocaml/xenopsd/xc/xenops_server_xen.ml

```diff
@@ -1162,7 +1162,7 @@ module VM = struct
       match persistent.ty with
       | Some (PV _) ->
           Memory.Linux.overhead_mib
-      | Some (PVinPVH _) ->
+      | Some (PVinPVH _ | PVH _ (* TODO: pvh *)) ->
```

Comment:
This is not consistent with the change in memory_check.ml, where the HVM overhead is used.